clientd3d/server.c

```diff
@@ -316,7 +316,7 @@ void ExtractDLighting(char **ptr, d_lighting *dLighting)
  * ExtractObject: Get an object_node from ptr, and increment
  *   ptr appropriately.  Place data in given object node.
  */
-void ExtractObject(char **ptr, object_node *item)
+void ExtractObject(char **ptr, object_node *item, bool includeLight = true)
```

Comment:
Yay, thank you!  But, it turns out the default value needs to go in the header file, due to weirdnesses about how default arguments work in C++:

https://stackoverflow.com/questions/14902129/default-parameters-in-h-and-cpp-files

Reply:
This was very interesting to learn about.  I moved the default value for the `includeLight` parameter to the header file declaration.